dra-evolution/pkg/api/capacity_types.go

```diff
@@ -69,6 +78,25 @@ type ResourcePoolSpec struct {
 	// them) empty pool.
 }
 
+// SharedCapacityGroup contains shared capacity values that are
+// part of a common set.
+//
+// For example, each physical, partitionable GPU would have one
+// SharedCapacityGroup.
+type SharedCapacityGroup struct {
+	Name string `json:"name"`
+
+	// SharedCapacity defines the sets of shared capacity consumable by
+	// devices in this SharedCapacityGroup.
+	//
+	// Must not have more than 16 entries.
+	//
+	// +listType=atomic
+	// +optional
+	SharedCapacity []SharedCapacity `json:"sharedCapacity,omitempty"`
```

Comment:
Maybe `items` is a better name so it's not stuttering so much.
```suggestion
	Items []SharedCapacity `json:"items,omitempty"`
```
I'd also say that its not optional. If you have one of these, it should have at least one item in it.